palettes.h

```diff
@@ -48,3 +56,768 @@ class RandomPalettes {
 	static std::set<GenerationTraits> random_traits();
 };
 
+struct Palettes {
+	struct Definition {
+		auto operator<=>(const Definition &other) const = default;
+
+		const std::wstring name;
+		const PaletteGroup group;
+		const PaletteData *data;
+	};
+
+	inline const static Definition Aemil = {
+		.name = L"aemil",
+		.group = PaletteGroup::Canon,
+		.data = new PaletteData {{
+			"#56eb8e",
+			"#84f5c3",
+			"#1d9550",
+			"#e29a56",

```

Comment:
Cool, right?

Reply:
:lkconcern:

Reply:
Well, uh... _I_ thought it was cool.

Reply:
My motivation was that it's easier to for example copy and paste color into and out of image editors. Plus it's way less ceremony to type. It is less safe because if you copy or type the hex code wrong it'll cause the program to blow up, but it should fail fast and be easy to spot the errors so I'm not _super_ worried about it.

Reply:
#41 This is exactly the kind of thing some run-time sanity checks would guard against.